mappings/net/minecraft/client/render/BufferBuilder.mapping

```diff
@@ -113,3 +113,10 @@ CLASS net/minecraft/class_287 net/minecraft/client/render/BufferBuilder
 		ARG 0 value
 		ARG 1 min
 		ARG 2 max
+	METHOD method_22622 rotate (Lnet/minecraft/class_1158;)V
+	METHOD method_22626 translate (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+	METHOD method_22629 newLayer ()V
```

Comment:
to fit with `removeLayer` should this maybe be `addLayer`?

Reply:
I think this might need a better name in general, It essentially clones the previous render buffer which can then be modified. But I will change it to add prefix.